### ISSUE TITLE
Check / fix table border color in dark mode

### ISSUE THREAD

**Issue by alya** [zulip/zulip-mobile#5871]
As a follow-up to https://github.com/zulip/zulip/issues/29856, we should see if it's worth tweaking the table border color in dark theme in the mobile app.

**Comment by gnprice** [comment]
Implementation notes:

The first step here is to see if the issue reproduces in mobile. If tables in dark mode already look fine and don't have this issue, then we're done.

If the issue does reproduce, then we should be able to fix it by just copying whatever CSS change is made to fix it in web.

**Comment by alya** [comment]
The webapp issue has been resolved in https://github.com/zulip/zulip/pull/29859.

**Comment by gnprice** [comment]
Cool, thanks for the update.

It looks like the key change was in this commit: https://github.com/zulip/zulip/pull/29859/commits/8a9fb047c3013d44749cddfe45a912c266e4e6dd
and changed the border color from `hsl(0deg 0% 0% / 20%)` to `hsl(0deg 0% 100% / 20%)`.

It kind of sounds like this might have been a regression caused in the first place by some other changes — it looks like what happened is that some CSS that was meant for a different part of the UI was too broad and consequently applied also to tables in Markdown content. So that suggests the issue might not exist in mobile. We'll find out as the first step in this issue, as mentioned above.

**Comment by alya** [comment]
I'm not sure this is a priority for the legacy app? In any case, removing the release goal label, as it's not specifically timely for 9.0.

**Comment by gnprice** [comment]
Yeah — I think as long as this isn't something we feel we're likely to highlight in the 9.0 announcement (and I suspect it isn't), it isn't a priority for the legacy app.

**Comment by YGsoc2025** [comment]
can i solve this problem @alya

**Comment by dineshkolhe1** [comment]
"Got it! Since this isn't a priority for the legacy app, I'll proceed with a solution that aligns with the current needs. Let me know if there are any specific guidelines or considerations you'd like me to keep in mind while working on this

**Comment by alya** [comment]
We aren't going to do this for the legacy app.
